CriteoPublisherSdk/Tests/UnitTests/Network/CR_BidRequestSerializerSwiftTests.swift

```diff
@@ -163,13 +164,48 @@ class CR_BidRequestSerializerSwiftTests: XCTestCase {
     XCTAssertEqual(publisher["ext"], expected)
   }
 
+  func testBodyWithCoppa() {
+
+    // case nil - undefined
+    coppa = nil
+    var body = generateBody()
+    XCTAssertFalse(body.keys.contains(CR_ApiQueryKeys.regs))
+
+    var regsDictionary: Dictionary<String, AnyHashable>
+    var coppaFlag: NSNumber
+
+    func unwrap(key: String, from dictionary: Dictionary<String, AnyHashable>) -> AnyHashable? {
```

Comment:
I don't understand what this method is doing. Isn't it just possible to access the key and let the runtime throw in case of missing key ?

Like: `(body['regs'] as! Dictionary<...>)['coppa'] should be true/false`

Reply:
It's a safe way to unwrap optionals from a dictionary. The benefit over the direct access it the custom message that is displayed if the unwrapping were to fail.